bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -51,6 +51,12 @@ cl::opt<bool>
                       cl::desc("Infer counts from stale profile data."),
                       cl::init(false), cl::Hidden, cl::cat(BoltOptCategory));
 
+cl::opt<unsigned> MatchedProfileThreshold(
+    "matched-profile-threshold",
```

Comment:
nit: the current name is quite ambiguous, suggest rename it to `stale-matching-min-matched-block` to be consistent with the convention used by other switches, and also more explicit. 

Suggestion for description: "Minimum number of exact match block for a function to be considered for profile inference."